### PR TITLE
information-schema: update the description for the tikv-region-status table

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Fix https://github.com/pingcap/docs-cn/issues/17482

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: information-schema/information-schema-tikv-region-status.md [hunk 1]
```diff
@@ -11,10 +11,12 @@ aliases: ['/docs-cn/dev/information-schema/information-schema-tikv-region-status
 {{< copyable "sql" >}}
```
Comment (qiancai) [inline information-schema/information-schema-tikv-region-status.md:12]:
```suggestion
```

File: information-schema/information-schema-tikv-region-status.md [hunk 2]
```diff
@@ -51,6 +54,9 @@ DESC tikv_region_status;
 * `IS_INDEX`：Region 数据是否是索引，0 代表不是索引，1 代表是索引。如果当前 Region 同时包含表数据和索引数据，会有多行记录，`IS_INDEX` 分别是 0 和 1。
 * `INDEX_ID`：Region 所属的索引的 ID。如果 `IS_INDEX` 为 0，这一列的值就为 NULL。
 * `INDEX_NAME`：Region 所属的索引的名称。如果 `IS_INDEX` 为 0，这一列的值就为 NULL。
+* `IS_PARTITION`：Region 所属的表是否为分区表。
+* `PARTITION_ID`：如果 Region 所属的表是否为分区表，显示 Region 所属的分区的 ID。
+* `PARTITION_NAME`：如果 Region 所属的表是否为分区表，显示 Region 所属的分区名。
```
Comment (qiancai) [inline information-schema/information-schema-tikv-region-status.md:59]:
```suggestion
* `PARTITION_NAME`：如果 Region 所属的表是否为分区表，显示 Region 所属的分区的名字。
```

**Review by glorv — APPROVED** [review]
LGTM

File: information-schema/information-schema-tikv-region-status.md [hunk 3]
```diff
@@ -51,6 +54,9 @@ DESC tikv_region_status;
 * `IS_INDEX`：Region 数据是否是索引，0 代表不是索引，1 代表是索引。如果当前 Region 同时包含表数据和索引数据，会有多行记录，`IS_INDEX` 分别是 0 和 1。
 * `INDEX_ID`：Region 所属的索引的 ID。如果 `IS_INDEX` 为 0，这一列的值就为 NULL。
 * `INDEX_NAME`：Region 所属的索引的名称。如果 `IS_INDEX` 为 0，这一列的值就为 NULL。
+* `IS_PARTITION`：Region 所属的表是否为分区表。
+* `PARTITION_ID`：如果 Region 所属的表是否为分区表，显示 Region 所属的分区的 ID。
+* `PARTITION_NAME`：如果 Region 所属的表是否为分区表，显示 Region 所属的分区的名字。
```
Comment (Oreoxmt) [inline information-schema/information-schema-tikv-region-status.md:59]:
```suggestion
* `PARTITION_ID`：如果 Region 所属的表为分区表，显示 Region 所属的分区的 ID。
* `PARTITION_NAME`：如果 Region 所属的表为分区表，显示 Region 所属的分区的名字。
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Pull request merged**